app/src/main/res/values-tr/strings.xml

```diff
@@ -21,15 +21,15 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple App Launcher - Find your apps quickly</string>
+    <string name="app_title">Basit Uygulama Başlatıcı - Uygulamalarınızı çabucak bulun</string>
```

Comment:
too long app name, it has to be max 50 characters long. If you cannot shorten it, just remove a part of it.

Reply:
I will close this pull request, it will be easier to open a new one than editing this